MobileFilter/sections/specific_web.txt

```diff
@@ -70,6 +70,7 @@ otakei.otakuma.net,nikkan-gendai.com,ranking.goo.ne.jp,m.hellporno.com,topics.sm
 !
 ! SECTION: Regular rules
 !
+motorauthority.com##.adv-spacer-top-wraper
```

Comment:
There are also few ads, social buttons, and subscriptions

<details><summary>Screenshot:</summary>

<img width="353" alt="Screenshot 2024-04-24 at 5 45 02 PM" src="https://github.com/AdguardTeam/AdguardFilters/assets/91964807/44e026d2-1f63-467a-a42e-e71c62a1d4e6">

</details><br/>


<details><summary>Screenshot:</summary>

<img width="357" alt="Screenshot 2024-04-24 at 5 45 45 PM" src="https://github.com/AdguardTeam/AdguardFilters/assets/91964807/ce583397-4e7a-4674-bc1c-6d3054db527f">

</details><br/>

<details><summary>Screenshot:</summary>

<img width="358" alt="Screenshot 2024-04-24 at 5 46 30 PM" src="https://github.com/AdguardTeam/AdguardFilters/assets/91964807/41de548f-0ade-451d-b081-9ff0ed515600">

</details><br/>

https://www.motorauthority.com/

<details><summary>Screenshot:</summary>

<img width="350" alt="Screenshot 2024-04-24 at 5 48 01 PM" src="https://github.com/AdguardTeam/AdguardFilters/assets/91964807/315bcdb3-2dbe-48d7-b7f0-4c86e68270c8">

</details><br/>

Reply:
Done